Endpoint of OSS should be with no `http://` scheme

## Enhancement
```
LOAD LABEL db0.labela5 (     DATA INFILE("'oss://starrocks-qa-test-data/xxxx")     INTO TABLE tbl_student     FORMAT AS "parquet"     (name, age)  ) WITH BROKER (   "fs.oss.accessKeyId"  =  "xxx",  "fs.oss.accessKeySecret"  =  "xxx", "fs.oss.endpoint"  =  "http://oss-cn-zhangjiakou.aliyuncs.com");
```
When we create broker load with `"fs.oss.endpoint"  =  "http://xxx"`, the load would fail.
```
         JobId: 15008
         Label: clickbench_csv_11123111201111111
         State: QUEUEING
      Progress: ETL:100%; LOAD:0%
          Type: BROKER
      Priority: NORMAL
      ScanRows: 0
  FilteredRows: 0
UnselectedRows: 0
      SinkRows: 0
       EtlInfo: NULL
      TaskInfo: resource:N/A; timeout(s):71342; max_filter_ratio:0.0
      ErrorMsg: type:ETL_RUN_FAIL; msg:Still retrying, error: listPath failed
    CreateTime: 2023-08-16 20:08:07
  EtlStartTime: NULL
 EtlFinishTime: NULL
 LoadStartTime: NULL
LoadFinishTime: NULL
   TrackingSQL: 
    JobDetails: {"All backends":{},"FileNumber":0,"FileSize":0,"InternalTableLoadBytes":0,"InternalTableLoadRows":0,"ScanBytes":0,"ScanRows":0,"TaskNumber":0,"Unfinished backends":{}}

```
The SR uses a wrong address `http://starrocks-qa-test-data.http://oss-cn-zhangjiakou.aliyuncs.com` to access the OSS service.
```
2023-08-16 20:12:00,769 WARN (pending_load_task_scheduler_pool-2|187) [LoadTask.exec():79] LOAD_JOB=15008, error_msg={Failed to execute load task}
com.starrocks.common.UserException: listPath failed
        at com.starrocks.fs.hdfs.HdfsFsManager.listPath(HdfsFsManager.java:1244) ~[starrocks-fe.jar:?]
        at com.starrocks.fs.hdfs.HdfsService.listPath(HdfsService.java:60) ~[starrocks-fe.jar:?]
        at com.starrocks.fs.HdfsUtil.parseFile(HdfsUtil.java:83) ~[starrocks-fe.jar:?]
        at com.starrocks.fs.HdfsUtil.parseFile(HdfsUtil.java:97) ~[starrocks-fe.jar:?]
        at com.starrocks.load.loadv2.BrokerLoadPendingTask.getAllFileStatus(BrokerLoadPendingTask.java:97) ~[starrocks-fe.jar:?]
        at com.starrocks.load.loadv2.BrokerLoadPendingTask.executeTask(BrokerLoadPendingTask.java:75) ~[starrocks-fe.jar:?]
        at com.starrocks.load.loadv2.LoadTask.exec(LoadTask.java:73) ~[starrocks-fe.jar:?]
        at com.starrocks.task.PriorityLeaderTask.run(PriorityLeaderTask.java:38) ~[starrocks-fe.jar:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_212]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_212]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_212]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_212]
        at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_212]
Caused by: java.io.IOException: ErrorCode : 2001, ErrorMessage :  [HostId]: http://oss-cn-zhangjiakou.aliyuncs.com [ErrorMessage]: Failed to connect Http service: http://starrocks-qa-test-data.http://oss-cn-zhangjiakou.aliyuncs.com/?max-keys=1&prefix=benchmark_data%2Fload_data%2Fclickbench%2Fcsv%2F [RequestId]:
        at com.aliyun.jindodata.call.JindoListCall.execute(JindoListCall.java:68) ~[jindo-sdk-4.6.8.jar:?]
        at com.aliyun.jindodata.oss.JindoOssFileSystem.listStatus(JindoOssFileSystem.java:217) ~[jindo-sdk-4.6.8.jar:?]
        at com.aliyun.jindodata.oss.JindoOssFileSystem.listStatus(JindoOssFileSystem.java:63) ~[jindo-sdk-4.6.8.jar:?]
        at org.apache.hadoop.fs.Globber.listStatus(Globber.java:128) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.Globber.doGlob(Globber.java:291) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.Globber.glob(Globber.java:202) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.FileSystem.globStatus(FileSystem.java:2225) ~[hadoop-common-3.3.6.jar:?]
        at com.starrocks.fs.hdfs.HdfsFsManager.listPath(HdfsFsManager.java:1208) ~[starrocks-fe.jar:?]
        ... 12 more
Caused by: com.aliyun.jindodata.api.spec.JdoException:  [HostId]: http://oss-cn-zhangjiakou.aliyuncs.com [ErrorMessage]: Failed to connect Http service: http://starrocks-qa-test-data.http://oss-cn-zhangjiakou.aliyuncs.com/?max-keys=1&prefix=benchmark_data%2Fload_data%2Fclickbench%2Fcsv%2F [RequestId]:
        at com.aliyun.jindodata.api.spec.JdoNativeResult.get(JdoNativeResult.java:58) ~[jindo-core-4.6.8.jar:?]
        at com.aliyun.jindodata.api.spec.protos.coder.JdolistDirectoryReplyDecoder.decode(JdolistDirectoryReplyDecoder.java:23) ~[jindo-core-4.6.8.jar:?]
        at com.aliyun.jindodata.api.JindoCommonApis.listDirectory(JindoCommonApis.java:90) ~[jindo-core-4.6.8.jar:?]
        at com.aliyun.jindodata.call.JindoListCall.execute(JindoListCall.java:65) ~[jindo-sdk-4.6.8.jar:?]
        at com.aliyun.jindodata.oss.JindoOssFileSystem.listStatus(JindoOssFileSystem.java:217) ~[jindo-sdk-4.6.8.jar:?]
        at com.aliyun.jindodata.oss.JindoOssFileSystem.listStatus(JindoOssFileSystem.java:63) ~[jindo-sdk-4.6.8.jar:?]
        at org.apache.hadoop.fs.Globber.listStatus(Globber.java:128) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.Globber.doGlob(Globber.java:291) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.Globber.glob(Globber.java:202) ~[hadoop-common-3.3.6.jar:?]
        at org.apache.hadoop.fs.FileSystem.globStatus(FileSystem.java:2225) ~[hadoop-common-3.3.6.jar:?]
        at com.starrocks.fs.hdfs.HdfsFsManager.listPath(HdfsFsManager.java:1208) ~[starrocks-fe.jar:?]
        ... 12 more
```

--- a/fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java
+++ b/fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java
@@ -956,7 +956,7 @@ public class HdfsFsManager {
         WildcardURI pathUri = new WildcardURI(path);
         String accessKey = loadProperties.getOrDefault(FS_OSS_ACCESS_KEY, "");
         String secretKey = loadProperties.getOrDefault(FS_OSS_SECRET_KEY, "");
-        String endpoint = loadProperties.getOrDefault(FS_OSS_ENDPOINT, "");
+        String endpoint = loadProperties.getOrDefault(FS_OSS_ENDPOINT, "").replaceFirst("^https?://", "");
         String disableCache = loadProperties.getOrDefault(FS_OSS_IMPL_DISABLE_CACHE, "true");
         String connectionSSLEnabled = loadProperties.getOrDefault(FS_OSS_CONNECTION_SSL_ENABLED, "false");
         // endpoint is the server host, pathUri.getUri().getHost() is the bucket

